contracts/atm/TLRToken.sol

```diff
@@ -179,8 +179,8 @@ contract TLRToken is
         require(account != address(0x0), "MINT_TO_ZERO_ADDRESS_NOT_ALLOWED");
         require(vestingCount[account] < _maxVestingPerWallet, "MAX_VESTINGS_REACHED");
         _beforeTokenTransfer(address(0x0), account, amount);
-        uint256 vestingId = vestingCount[account]++;
         vestingCount[account] += 1;
+        uint256 vestingId = vestingCount[account];
```

Comment:
I remember seeing this modified in another PR. Just making sure we are in sync.

Reply:
Yes, I added it into the TLRToken PR after speaking with Juan

Reply:
Thanks for noticing this @passabilities! Yes, as Syed said this was implemented also on @sypeer 's PR so it's fixed